lib/const.rb

```diff
@@ -224,3 +224,5 @@
 # libssp is in the libssp package
 # libatomic is in the gcc package
 CREW_ESSENTIAL_FILES = %x[LD_TRACE_LOADED_OBJECTS=1 #{CREW_PREFIX}/bin/ruby].scan(/\t([^ ]+)/).flatten
+
+CREW_DISABLE_ENV_OPTION = ENV['CREW_DISABLE_ENV_OPTION']
```

Comment:
Combine with `CREW_ENV_OPTIONS` to make these changes globally (some packages are using `CREW_ENV_OPTIONS` in `./configure`)?
```ruby
unless CREW_DISABLE_ENV_OPTIONS
  CREW_ENV_OPTIONS_HASH = ...
else
  CREW_ENV_OPTIONS_HASH = { "CREW_DISABLE_ENV_OPTION" => '1' }
end
```

Reply:
(by that way we don't have to modify `self.system`)

Reply:
Brilliant. Yes. Done.